Description: Remove references to sun proprietary dtds.

Sun dtds are shipped within javahelp-filters. They are being removed
from tarball, and here we avoid looking for them locally.

Forwarded: no
Author: Tiago Saboga <tiagosaboga@gmail.com>
Last-Update: 2011-06-07
--- a/addons/javahelp-filters/src/org/omegat/filters2/xml2/XMLHandler2.java
+++ b/addons/javahelp-filters/src/org/omegat/filters2/xml2/XMLHandler2.java
@@ -143,28 +143,6 @@
         write("</"+qName+">");
     }
 
-    /** Resolve an external entity to work offline or behind proxy. */
-    public InputSource resolveEntity(String publicId, String systemId) 
-            throws IOException, SAXException
-    {
-        String orig = systemId;
-        int lastslash = systemId.lastIndexOf('/');
-        if( lastslash>=0 )
-        {
-            String name = orig.substring(lastslash+1);
-            String res = "/org/omegat/filters2/javahelp/resources/"+name;
-            InputStream stream = getClass().getResourceAsStream(res);
-            return new InputSource(stream);
-        }
-        return null;
-    }
-    /** The older {@link #resolveEntity} method is overridden to call this one. */
-    public InputSource resolveEntity(String name, String publicId, String baseURI, String systemId) 
-            throws SAXException, IOException
-    {
-        return resolveEntity(publicId, systemId);
-    }
-
     /** Receive notification of ignorable whitespace in element content.*/
     public void ignorableWhitespace(char[] ch, int start, int length) throws SAXException
     {
